### ISSUE TITLE
[Improvement] Custom step_definitions path

### ISSUE THREAD

**Issue by simplyageek** [tomkadwill/atom-cucumber-autocomplete#28]
I'd suggest adding a new config value to settings, which would hold the path for the step definition files (to scan). Maybe I don't like to have my steps under /features/step_definitions

**Comment by tomkadwill** [comment]
Hi @beniamin-kis, you can change the path that the plugin looks for under settings. Is that what you are looking for?

**Comment by simplyageek** [comment]
No, under settings I can only set the path for the feature files.
But my project structure is:
[root]/src/features/login.feature etc.
[root]/src/steps/givens.js etc.

Your plugin scans for steps under the features path only. So if I don't have my steps in
[root]/src/features/step_definitions/*.js
then it can't find them.

My suggestion was a second path configuration under settings, which would hold the path for the steps files.

Check this line:
https://github.com/tomkadwill/atom-cucumber-autocomplete/blob/master/lib/provider.js#L60

See the hardcoded path value? That should be configurable.

Same for line #48
